torchinductor/config.py

```diff
@@ -16,6 +16,8 @@
 # enable certain approximation algorithms
 approximations = False
 
+# select compiler backend for cuda
+cuda_backend = 'Jiteraor'
```

Comment:
In the final version let's leave triton as the default

Reply:
Yes, will do.